src/test/ui/parser/increment-autofix.stderr

```diff
@@ -0,0 +1,89 @@
+error: Rust has no postfix increment operator
+  --> $DIR/increment-autofix.rs:5:6
+   |
+LL |     i++;
+   |      ^^ not a valid postfix operator
+   |
+help: use `+= 1` instead
+   |
+LL |     { let tmp = i; i += 1; tmp };
+   |     +++++++++++  ~~~~~~~~~~~~~~~
+help: or, if you don't need to use it as an expression, change it to this
```

Comment:
```suggestion
help: or, if you don't need to use the incremented value
```
I think this might be a little bit clearer, avoids terminology like "expression" which might not be obvious to a beginner.

Reply:
Yeah, I was a bit concerned about saying "expression". However, "the incremented value" is actually not quite right because the suggestion yields the pre-increment value. What about this?

```suggestion
help: or, if you don't need to use the pre-increment value
```

or this?

```suggestion
help: or, if you don't need to use the old value
```

Reply:
Either of these are good, I don't mind which you pick, I think the first is clearer but it also has more terminology